tests/Integration/EventListener/DoctrineEventSubscriberTest.php

```diff
@@ -162,7 +162,7 @@ public function testPreRemoveWithObjectId(): void
      */
     private function waitForAllTasks(): void
     {
-        $firstTask = $this->client->getTasks()['results'][0];
+        $firstTask = $this->client->getTasks()->getResults()[0];
```

Comment:
What is the new return type of `getTasks`, it was an array before, right?

Reply:
Intrinsically is the same behavior but the correct way to access this value is through the gettter :)

Reply:
Perfect! thanks!